Add @datagouv/components CI (#407)
datagouv/udata-front: udata_front/theme/gouvfr/datagouv-components/README.md

## env variables

You can customize how this package work with env variables during build and runtime (with `setupComponents` function).

> ## Publish a new version

Reviewer: Thank you for this new section :pray:

`datagouv-components` is published as `@datagouv/components` on [NPM](https://www.npmjs.com/package/@datagouv/components).

There are 2 differents channels (called tag in NPM) of versions :
- `latest`, the default one, for the releases based on git tags named following the convention `@datagouv/components@version`
- `dev`, for our development versions based on `master`

`dev` versions are done automatically and there is nothing to do.
For our releases, a manual update of the version number is necessary. To do so, you can run :

```
npm run update-version -- NEW_VERSION
```

> You should use `major`, `minor`, `patch` to let NPM compute the next version number.

Reviewer: ```suggestion
`NEW_VERSION` should be either `major`, `minor` or `patch` to let NPM compute the next version number.
```
